### PR TITLE
[SHELL32] CDefView: Implement SFVM_GETCOMMANDIR callback

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Implementing missing folder view callbacks...
JIRA issue: [CORE-19616](https://jira.reactos.org/browse/CORE-19616)

## Proposed changes

- In `CDefView::InvokeContextMenuCommand`, call `SFVM_GETCOMMANDDIR` callback and store the directory into `CMINVOKECOMMANDINFOEX` structure.

## TODO

- [x] Do build.

### CODE REVIEW

File: dll/win32/shell32/CDefView.cpp [hunk 1]
```diff
@@ -1582,6 +1582,18 @@ HRESULT CDefView::InvokeContextMenuCommand(CComPtr<IContextMenu>& pCM, LPCSTR lp
         cmi.ptInvoke = *pt;
     }
 
+    WCHAR szDirW[MAX_PATH];
+    CHAR szDirA[MAX_PATH];
+    szDirW[0] = 0;
```
Comment (HBelusca) [inline dll/win32/shell32/CDefView.cpp:1587]:
```suggestion
    WCHAR szDirW[MAX_PATH] = L"";
    CHAR szDirA[MAX_PATH];
```

File: dll/win32/shell32/CDefView.cpp [hunk 2]
```diff
@@ -1582,6 +1582,18 @@ HRESULT CDefView::InvokeContextMenuCommand(CComPtr<IContextMenu>& pCM, LPCSTR lp
...
+    if (SUCCEEDED(_DoFolderViewCB(SFVM_GETCOMMANDDIR, _countof(szDirW), (LPARAM)szDirW)) &&
+        szDirW[0] != UNICODE_NULL)
```
Comment (HBelusca) [inline dll/win32/shell32/CDefView.cpp:1589]: or
```suggestion
        *szDirW != UNICODE_NULL)
```
or shorter,
```suggestion
        *szDirW)
```

File: dll/win32/shell32/CDefView.cpp (hunk 1 above)
Comment (whindsaks) [reply dll/win32/shell32/CDefView.cpp]: `szDirW[0] = 0;` is a single instruction, `[MAX_PATH] = L"";` is a function call to `memset`!

**Review by HBelusca — APPROVED** [review]

File: dll/win32/shell32/CDefView.cpp (hunk 1 above)
Comment (katahiromz) [reply dll/win32/shell32/CDefView.cpp]: Agree to @whindsaks.

**Pull request merged**